content/en/contribute/medic/team.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Team Organization"
+linkTitle: "Team Organization"
+weight: 1
+description: >
+   Product Team Organization at Medic
+---
+
+
+### Product Team Leadership 
+|Title|Role|
+|--|--|
+|Chief Product Officer|Leads & aligns the team to achieve ambitious organizational goals|
+|Senior Director of Technology|Ensures that digital health systems and internal processes are built on cohesive set of technologies that can grow in a sustainable manner|
+|Director of Product Development|Directs the processes and teammates to build and maintain the Community Health Toolkit|
+
+### Focused Groups
+[Focused Groups](/contribute/medic/product-development-process/focused-groups)
```

Comment:
As per comment above, we shouldn't use the shorthand here. Also, I imagine this may have just been a placeholder, but we should add a bit of text here to avoid just repeating the title with the link.

```suggestion
### Focused Working Groups
[Focused Working Groups](/contribute/medic/product-development-process/focused-groups)
```